lib/src/key/cl.rs

```diff
@@ -13,17 +13,13 @@ pub struct Cl {
 	pub nd: Uuid,
```

Comment:
I guess `nd == node`?

If i'm not mistaken, the compiler takes care of abbreviating long names, so I vote for longer variable names in the code 😄  I broke the tradition in #2098 💥